ModernWpf/Media/Animation/PointerUpThemeAnimation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Media.Animation;
+using ModernWpf.Controls.Primitives;
+
+namespace ModernWpf.Media.Animation
+{
+    /// <summary>
+    /// Represents the preconfigured pointer up animation that applies to controls when
+    /// they are pressed.
```

Comment:
Typo error : have to change 'when they are pressed' to 'after they are released'